### PR TITLE
envvault 1.0.0 (new formula)

### PR DESCRIPTION
**Pull request by sanjithmsk**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by sanjithmsk** [comment]
Secure, scalable, and collaborative environment variable management. Built for developers, by developers

**Comment by sanjithmsk** [comment]
what i have to do next step to add my package to home brew

**Review by sanjithmsk — COMMENTED** [review]
Secure, scalable, and collaborative environment variable management. Built for developers, by developers

**Comment by sanjithmsk** [comment]
running all tests.

**Comment by SMillerDev** [comment]
> what i have to do next step to add my package to home brew

It needs to pass all checks

**Review by SMillerDev — COMMENTED** [review]
How is this different from https://github.com/envault/envault?

File: Formula/e/envvault.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+
+# envvault.rb
+# developed by @Fahid Mohammad
+# version 1.0.0
+# date 2024-12-17
+
+class Envvault < Formula
+    desc "Secure, scalable, and collaborative environment variable management. Built for developers, by developers"
+    homepage "https://envvault.com/"
+    url "https://envvault.com/releases/envvault-1.0.0.tar.gz"
```
Comment (carlocab) [inline Formula/e/envvault.rb:10]: This tarball contains a precompiled binary. Formulae in Homebrew/core should build from source.

https://docs.brew.sh/Acceptable-Formulae#we-dont-like-binary-formulae

**Review by carlocab — CHANGES_REQUESTED** [review]

**Pull request closed without merging**